### PR TITLE
fix platform info

### LINKED ISSUE #2197: "Works with ..." text is missing from Desktop compatible Android addons
Prerequisites:
Build identifier: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:45.0) Gecko/20100101 Firefox/45.0

Steps to reproduce:
1.Go to https://addons-dev.allizom.org/en-US/android/
2.Hover with mouse over some Android addons which are compatible with Desktop too

Expected results:
There is a text saying that the addon is working with a range of Firefox for Android versions

Actual results:
This text is missing

Please see video for this bug : http://screencast.com/t/lMrZpCCbxj3W

### LINKED ISSUE #2314: Compatibility and version info look like a button but aren't
Steps to reproduce:
1. Load details page for an incompatible add-on in AMO-dev : https://addons-dev.allizom.org/en-US/firefox/addon/nasa-night-launch-test/?src=search
2. Observe the compatibility info

Expected results:
The info has not display or layout issues.

Actual results:
The info look like a button.

Notes/Issues:
This is reproducing for Firefox addons, SeaMonkey add-ons, Thunderbird add-ons using FF45 and Chrome.
Issue is reproducing on AMO-stage , dev and prod.
Please see the screenshots:
firefox
chrome
android
thunder
seamonkey

### LINKED ISSUE #2321: Compatibility info incorrectly displayed
**Steps to reproduce**
1. Navigate to https://addons.allizom.org/en-US/
2. Search for an incompatible add-on (i.e ezstylist )
3. Hover over the add-on.

**Expected Results**
Compatibility info is correctly displayed.

**Actual Results**
There is no space before “View other versions” link. 

**Additional notes:**
- This issue reproduces on AMO-stage using Firefox 48.0a1 (2016-04-11) under Windows 10 64-bit.
- It is not reproducible on add-ons details page (see screenshot)

- Please see the screenshot for this issue:
vv

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #2314 
Fixes #2197
Fixes #2321

Less button-like notice.

tap_translate____add-ons_for_firefox_for_android

Restored platform info:

add-ons_for_firefox_for_android

Fixed link space:

phony____add-ons_for_firefox_for_android

### CODE REVIEW

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -1586,7 +1586,21 @@ h3.author .transfer-ownership {
...
     }
   }
+}
+
+.extra .button.disabled.not-available {
+  background-color: #fff;
+  border: 1px solid #ebebeb;
+  border-radius: 0;
+  box-shadow: none;
+  color: #333;
```
Comment (tofumatt) [inline static/css/restyle/restyle.less]: Nit: this could be `@default-font-color`

File: static/css/restyle/restyle.less [hunk 2]
```diff
@@ -1655,18 +1669,45 @@ h3.author .transfer-ownership {
   text-shadow: none;
 }
 
+.addon.hovercard,
 .items .item.incompatible {
+
```
Comment (tofumatt) [inline static/css/restyle/restyle.less]: Eventually we'll need an issue to convert all selectors to have this newline or not ^_^

File: static/css/restyle/restyle.less [hunk 3]
```diff
@@ -1655,18 +1669,45 @@ h3.author .transfer-ownership {
...
   .extra .button.not-available {
     display: none;
     float: right;
   }
 
+  .extra {
+    margin-top: 0 !important;
```
Comment (tofumatt) [inline static/css/restyle/restyle.less]: I'm gonna trust you that these are totally necessary :wink:

File: static/css/restyle/restyle.less [hunk 4]
```diff
@@ -1655,18 +1669,45 @@ h3.author .transfer-ownership {
...
+  }
+
   &:focus .extra .button.not-available,
   &:hover .extra .button.not-available {
+    display: block !important;
```
Comment (tofumatt) [inline static/css/restyle/restyle.less]: THE `!important`, IT BURNS

**Comment by tofumatt** [comment]
Looks much better :+1: 

r+wc

File: static/css/restyle/restyle.less (hunk 2 above)
Comment (muffinresearch) [inline static/css/restyle/restyle.less]: Yeah I like to let things breath.

File: static/css/restyle/restyle.less (hunk 3 above)
Comment (muffinresearch) [inline static/css/restyle/restyle.less]: Thanks for questioning that - I was able to remove that one at least.

**Pull request merged**